disnake/utils.py

```diff
@@ -1200,6 +1201,72 @@ def resolve_annotation(
     return evaluate_annotation(annotation, globalns, locals, cache)
 
 
+def unwrap_function(function: Callable[..., Any]) -> Callable[..., Any]:
+    partial = functools.partial
+    while True:
+        if hasattr(function, "__wrapped__"):
+            function = function.__wrapped__
+        elif isinstance(function, partial):
+            function = function.func
+        else:
+            return function
+
+
+def _get_function_globals(function: Callable[..., Any]) -> Dict[str, Any]:
+    unwrap = unwrap_function(function)
+    try:
+        return unwrap.__globals__
+    except AttributeError:
+        return {}
+
+
+_inspect_empty = inspect.Parameter.empty
+
+
+def get_signature_parameters(
+    function: Callable[..., Any], globalns: Optional[Dict[str, Any]] = None
```

Comment:
```suggestion
    function: Callable[..., Any], globals: Optional[Dict[str, Any]] = None
```
Is it a typo? It seems to appear in all files modified by this PR

Reply:
`globalns` (global namespace) is generally used to not shadow `globals`, so I don't think this is a typo ^^

Reply:
Ok, makes sense. It confused me because some functions / parsers don't follow this naming pattern